validation/src/main/scala/hmda/validation/SubmissionLarStats.scala

```diff
@@ -0,0 +1,58 @@
+package hmda.validation
+
+import akka.actor.{ ActorRef, ActorSystem, Props }
+import hmda.model.fi.SubmissionId
+import hmda.persistence.messages.CommonMessages.{ Command, Event, GetState }
+import hmda.persistence.messages.events.processing.CommonHmdaValidatorEvents.LarValidated
+import hmda.persistence.model.HmdaPersistentActor
+import hmda.validation.ValidationStats.{ AddSubmissionStats, SubmissionStats }
+
+object SubmissionLarStats {
+  def name = "SubmissionStats"
+
+  case class CountLarsInSubmission() extends Command
+  case class SubmissionStatsUpdated(total: Int) extends Event
+
+  def props(submissionId: SubmissionId): Props = Props(new SubmissionLarStats(submissionId))
+
+  def createSubmissionStats(system: ActorSystem, submissionId: SubmissionId): ActorRef = {
+    system.actorOf(SubmissionLarStats.props(submissionId))
+  }
+
+  case class SubmissionLarStatsState(totalLars: Int = 0) {
+    def updated(event: Event): SubmissionLarStatsState = event match {
+      case SubmissionStatsUpdated(total) =>
+        SubmissionLarStatsState(total)
+    }
+  }
+}
+
+class SubmissionLarStats(submissionId: SubmissionId) extends HmdaPersistentActor {
```

Comment:
I'm having trouble understanding this actor. The `HmdaFileValidationState` already holds every lar that was validated for the submission, so it seems like this actor's job could just be done in the `HmdaFileValidator`. 

Is this to prevent overloading that actor with too many responsibilities? Or other purpose I'm missing?